### DISCUSSION TITLE
Does Uninstalling Armory Crate break Auto Updates?

### DISCUSSION THREAD

**Discussion by sr2echa** [seerge/g-helper#1782] [Q&A]
does uninstalling Armory Crate also remove the auto-update of bios and other drivers during Windows Update? Or will if auto update the stuff like before? 
Cause G-helper has a separate panel for checking and manually updating the drives, but not essential if it wont break the auto-update system.

**Comment by seerge** [comment]
@sr2echa Armoury doesn't update anything besides it's own components.

Driver updates are usually handled by MyASUS.

Windows Update has nothing to do neither with Armoury nor MyAsus and it will keep going as it was going before
